src/utils.js

```diff
@@ -982,17 +982,28 @@ function checkWebFlags (flag) {
  *
  * @param {string} fullName the full action name prefixed with the package, e.g. `pkg/action`
  * @param {ManifestAction} manifestAction the action object as parsed from the manifest
+ * @param {object} [options={}] options object to store params
+ * @param {boolean} [options.actionCode] param: if false, skips action code deployment
  * @returns {OpenWhiskEntitiesAction} the action entity object
  */
-function createActionObject (fullName, manifestAction) {
+function createActionObject (fullName, manifestAction, options = {}) {
+  /// c. options param missing actionCode property
+  const optionsParam = cloneDeep(options)
+  if (optionsParam.actionCode !== false) {
```

Comment:
Instead of this check we can add actionCode = true as default value for options param

Reply:
Hi @sandeep-paliwal . So good to have you back! :)
Thanks for the review. Initially, I added actionCode = true as the default value for options param but then Shaz rightly pointed out that there could be a case when the API is called with empty options object:
`createActionObject (fullName, manifestAction,{})`
In that case default value wouldn't apply. 
Link to the above conversation: https://github.com/adobe/aio-lib-runtime/pull/98#discussion_r891558501